### PR TITLE
Feat/weather

### PR DESCRIPTION
**Pull request by Speedfan94**
Add test NN and weather features

### CODE REVIEW

**Review by MaxStroh — APPROVED** [review]
Looks good :) 👍

File: nextbike/testing/nn_testing.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import pandas as pd
+from tensorflow import keras
+from tensorflow.keras import layers
+from sklearn.linear_model import LinearRegression
+from sklearn import metrics
+from .. import io
+
+
+def test_neuralnetwork_model(p_components, p_y_train):
+    # p_components=p_components.iloc["1", "2"]
```
Comment (weisske) [inline nextbike/testing/nn_testing.py:11]: Docstring missing :-P

File: nextbike/datapreparation/add_weather_data.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def add_weather(df_trips):
+    """Adds the weather data depending on the date and time to the trips DataFrame
+
+        Args:
+            df_trips (DataFrame): DataFrame from cli.py which contains already the cleaned trips
+        Returns:
+            df_trips (DataFrame): DataFrame, which contains the trips with the weather data
+        """
```
Comment (weisske) [inline nextbike/datapreparation/add_weather_data.py]: Rethink weather. 
Discussion during session.

**Review by weisske — APPROVED** [review]
Solved all comments

**Pull request merged**